xbmc/cores/AudioEngine/Sinks/AESinkWASAPI.cpp

```diff
@@ -771,7 +771,28 @@ bool CAESinkWASAPI::InitializeExclusive(AEAudioFormat &format)
       wfxex.Format.nChannels = noOfCh;
       int res = desired_map & wfxex.dwChannelMask;
       if (res != desired_map)
-        continue; // output channel layout doesn't match input channels
+      {
+        bool skip = true;
+        if (requestedChannels == 6 && wfxex.Format.nChannels == 6)
+        {
+          // For old sound cards it might look as follows
+          // Works: Channel Layout: FL, FR, FC, LFE, BL, BR
+          // Does not work: FL, FR, FC, LFE, SL, SR
+          // in the mask comparison SL and SR remain as driver only
+          // returns BL and BR channel
+          CLog::LogF(LOGDEBUG, "Fallback Matching Res: {} SL|SR {} BL|BR", res,
+                     (int)(SPEAKER_SIDE_LEFT | SPEAKER_SIDE_RIGHT),
+                     (int)(SPEAKER_BACK_LEFT | SPEAKER_BACK_RIGHT));
```

Comment:
Seems this log line has 3 parameters and only 2 placeholders? 

`"Fallback Matching Res: {} SL|SR {} BL|BR {}"`  ??

Also cast to int not seems necessary as SPEAKER_SIDE_LEFT / RIGHT are also ints

Reply:
Yes, sorry for that. Blind coding without runtime testing ... is not the best thing to keep jenkins resources low ;-). Thanks.